core/vm/gas_table.go

```diff
@@ -62,6 +62,7 @@ func memoryGasCost(mem *Memory, newMemSize uint64) (uint64, error) {
 // CODECOPY (stack position 2)
 // EXTCODECOPY (stack position 3)
 // RETURNDATACOPY (stack position 2)
+// MCOPY (stack position 2)
```

Comment:
please place this in the same order, below codecopy

Reply:
> please place this in the same order, below codecopy

i'm not sure i understood your comment - did you mean like this? https://github.com/ethereum/go-ethereum/pull/26181/commits/175802b730541e5330ed4aa1982180047e220350

Reply:
yes exactly thank you